src/Moryx.ControlSystem/VisualInstructions/IVisualInstructor.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0
 
 using Moryx.AbstractionLayer.Resources;
+using System;
 
 namespace Moryx.ControlSystem.VisualInstructions
 {
```

Comment:
Based o the discussion in #37 would it be to much to ask of you whether you could add the params and return xml documentation for the methods in here? What is for example the return value of the display method? 😅
And while you are at it, the explenation you gave in the same thread on what the IVisualInstructor and what the IWorkerSupport Facade are good for, do you think you could put that somehow into the interface xml comments? 🤔